English_transcripts/school/Epic Quest/sharedAssets/P_C_M_ENG_L3_AU_numline_dota_20_beg.txt

```diff
@@ -0,0 +1 @@
+You can use a number line to identify the numbers even if the number line does not begin with 0. This number line goes from number 5.
```

Comment:
You will not have a full stop here, since this statement is merged with another statement.